projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -18,6 +18,59 @@ import React from 'react';
  */
 import styles from './styles.module.scss';
 
+const ConnectionItem = props => {
+	const { connections } = window.jetpackSocialInitialState;
+
+	return props.connectionIds.map( connectionId => {
+		const connection = connections[ props.provider ][ connectionId ];
+		return (
+			<div
+				key={ connectionId + Date.now().toString( 36 ) + Math.random().toString( 36 ).substr( 2 ) }
```

Comment:
I think we should just make this `connectionId`, as otherwise the key will be changing on each render, and we won't get the benefit

Reply:
Got you, done!